doozerlib/cli/images_streams.py

```diff
@@ -608,6 +608,25 @@ def images_upstreampulls(runtime):
     print(yaml.dump(retdata, default_flow_style=False, width=10000))
 
 
+def align_issue_with_pr(github_client: Github, pr_url: str, issue: str):
+    """
+    Aligns an existing Jira issue with a PR. Put the issue number in the title of the github pr.
+    Args:
+        github_client: A github client object for reuse
+        pr_url: The URL of the PR to align with
+        issue: JIRA issue key
+    """
+    pr_info = pr_url.split('/')
+    source_repo = github_client.get_repo(f'{pr_info[-4]}/{pr_info[-3]}')
+    pr = source_repo.get_pull(int(pr_info[-1]))
+    if issue in pr.title:  # the issue already in pr title
+        return
+    elif "OCPBUGS" in pr.title:  # another issue is in pr title, add comment
+        pr.create_issue_comment(f"A JIRA issue [{issue}](https://issues.redhat.com/browse/{issue}) is also related to this pr when another OCPBUGS was in title.")
```

Comment:
Think this potentially leads to infinite amount of comments. Can this be limited?

Reply:
add a check

Reply:
Think the comment is not very clear. 

Maybe something like:
>ART wants to connect issue OCPBUGS-123 to this PR, but found it is currently hooked up to OCPBUGS-456. Please consult with #aos-art if it is not clear what there is to do.

Reply:
Unresolve. Comment could see improvements imho

Reply:
comment message is updated